core/src/main/scala/laserdisc/protocol/RESP.scala

```diff
@@ -254,7 +254,7 @@ sealed trait RESPCodecs { this: RESPBuilders =>
         override final def encode(bits: BitVector): Attempt[BitVector] = Attempt.successful(bits ++ crlf)
         override final def decode(bits: BitVector): Attempt[DecodeResult[BitVector]] =
           bits.bytes.indexOfSlice(crlfBytes, size) match {
-            case -1 => Attempt.failure(Err("Does not contain 'CRLF' termination bytes."))
+            case -1 => Attempt.failure(Err(s"Does not contain 'CRLF' termination bytes. Vector content: ${bits.toByteArray map (_.toChar) mkString ""}"))
```

Comment:
maybe `new String(bits.toByteArray, UTF_8)`? Also, maybe worth replacing `\r\n` with escaped sequence like `.replaceAll("\r\n", "\\\\r\\\\n")`?

Reply:
Bulk strings are binary-safe, so if you want a string that's always printable, maybe one of the representations (hex encoding?) from scodec?

Reply:
Also, if the data is quite large, the most interesting information is the last few bytes, not the whole block.